charts/incubator/plextraktsync/templates/_cronjob.tpl

```diff
@@ -1,52 +1,38 @@
 {{/* Define the cronjob */}}
 {{- define "plextraktsync.cronjob" -}}
-{{- $jobName := include "tc.common.names.fullname" . }}
-
----
-apiVersion: batch/v1
-kind: CronJob
-metadata:
-  name: {{ printf "%s-cronjob" $jobName }}
-  labels:
-    {{- include "tc.common.labels" . | nindent 4 }}
-spec:
-  schedule: "{{ .Values.plexTraktSync.schedule }}"
-  concurrencyPolicy: Forbid
-  {{- with .Values.cronjob.failedJobsHistoryLimit }}
-  failedJobsHistoryLimit: {{ . }}
-  {{- end }}
-  {{- with .Values.cronjob.successfulJobsHistoryLimit }}
-  successfulJobsHistoryLimit: {{ . }}
-  {{- end }}
-  jobTemplate:
-    metadata:
-    spec:
-      template:
-        metadata:
-        spec:
-          securityContext:
-            runAsUser: 0
-            runAsGroup: 0
-          restartPolicy: Never
-          containers:
-            - name: {{ .Chart.Name }}
-              securityContext:
-                privileged: false
-                readOnlyRootFilesystem: false
-                allowPrivilegeEscalation: false
-                runAsNonRoot: false
-                capabilities:
-                  drop:
-                    - ALL
-              image: "{{ .Values.image.repository }}:{{ .Values.image.tag }}"
-              command:
-                - plextraktsync
-                {{- if .Values.plexTraktSync.task }}
-                - {{ .Values.plexTraktSync.task }}
-                {{- else }}
-                - sync --sync=all
-                {{- end }}
-              resources:
-{{ toYaml .Values.resources | indent 16 }}
-
+enabled: true
+type: "CronJob"
+schedule: "{{ .Values.plextraktsync.schedule }}"
+podSpec:
+  restartPolicy: Never
+  containers:
+    cron:
+      enabled: true
+      primary: true
+      imageSelector: "image"
+      securityContext:
+        runAsNonRoot: false
+        readOnlyRootFilesystem: false
+        runAsUser: 0
+        runAsGroup: 0
+        capabilities:
+          drop:
+            - ALL
+      command:
+          - /bin/bash
+          - -c
+          - |
+            plextraktsync
+            {{- if  .Values.plextraktsync.task }}
+              {{ .Values.plextraktsync.task }}
+            {{- else }}
+              sync --sync=all
+            {{- end }}
```

Comment:
Uhm, this will execute `plextraktsync` once this is finished
it will execute either the task defined or `sync --sync=all` 
(The sync will be the linux shell command, not the subcommand of plextrakt)

Either use a yaml list as it was before or construct the command in one line

Reply:
like this?

```yaml
            {{- if  .Values.plextraktsync.task }}
              plextraktsync {{ .Values.plextraktsync.task }}
            {{- else }}
              plextraktsync sync --sync=all
            {{- end }}
```